Source/WebCore/platform/gtk/GtkUtilities.cpp

```diff
@@ -141,13 +142,16 @@ WallTime wallTimeForEvent(const GdkEvent* event)
 
 String defaultGtkSystemFont()
 {
-    GUniqueOutPtr<char> fontString;
-    g_object_get(gtk_settings_get_default(), "gtk-font-name", &fontString.outPtr(), nullptr);
+    auto fontName = SystemSettingsGLib::singleton().fontName();
+    if (!fontName || fontName->isEmpty())
+        return "Sans"_s;
+
     // We need to remove the size from the value of the property,
     // which is separated from the font family using a space.
-    if (auto* spaceChar = strrchr(fontString.get(), ' '))
+    char* fontString = fontName->utf8().mutableData();
+    if (auto* spaceChar = strrchr(fontString, ' '))
         *spaceChar = '\0';
-    return String::fromUTF8(fontString.get());
+    return String::fromUTF8(fontString);
```

Comment:
You can do this with WTFString API now and avoid utf8 back and froth conversions